tests/cpp-empty-test/Classes/AppDelegate.cpp

```diff
@@ -56,10 +56,14 @@ bool AppDelegate::applicationDidFinishLaunching()
     // Initialize director
     auto director = Director::getInstance();
     auto glview = director->getOpenGLView();
+    auto fileutils = FileUtils::getInstance();
     if(!glview) {
         glview = GLViewImpl::create("Cpp Empty Test");
         director->setOpenGLView(glview);
     }
+    std::vector<std::string> icons;
+    fileutils->listFilesRecursively("icons", &icons);
+	glview->setIcon(icons);
```

Comment:
here is a mistake, should use space not Tab

Reply:
There is already some TABs (except this) in this file. Should I replace them all?

Reply:
yep, replace them all, thanks for doing this.

Reply:
Done. Say thanks to my code editor!